pkg/provider/onepassword/onepassword.go

```diff
@@ -499,7 +500,12 @@ func (provider *ProviderOnePassword) findItem(name string) (*onepassword.Item, e
 			return provider.client.GetItem(name, vault.ID)
 		}
 
-		items, err := provider.client.GetItemsByTitle(name, vault.ID)
+		var items []onepassword.Item
+		err = retryOn403(func() error {
```

Comment:
I think line 500 is missing a wrap too. I think it might be safer if you introduce a client wrapper that automatically injects the retry logic. WDYT?

Reply:
Yah, might be better.